postings/tests.py

```diff
@@ -1,3 +1,105 @@
-from django.test import TestCase
+import json
 
-# Create your tests here.
+from django.test     import TestCase, Client
+from postings.models import *
+from users.models    import User
+from my_settings     import SECRET_KEY, ALGORITHM
```

Comment:
```suggestion
from django.test     import TestCase, Client

from postings.models import *
from users.models    import User
from my_settings     import SECRET_KEY, ALGORITHM
```
* 대신에 사용하는 모델을 import 해주세요.